python/lsst/daf/butler/utils.py

```diff
@@ -134,6 +134,33 @@ def globToRegex(expressions: str | EllipsisType | None | list[str]) -> list[str
     return results
 
 
+def has_globs(expressions: str | EllipsisType | None | Iterable[str]) -> bool:
+    """Determine if the expressions have any glob wildcard characters.
+
+    If a stand-alone '``*``' is found in ``expressions``, or expressions is
+    empty or `None`, then the special value ``...`` will be returned,
+    indicating that any string will match.
```

Comment:
This looks like it's copied from `globToRegex` but it probably does not apply here?